seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hudi-e2e/src/test/resources/fake_to_hudi.conf

```diff
@@ -46,7 +46,11 @@ source {
 
 sink {
   Hudi {
+    op_type="UPSERT"
     table_dfs_path = "/tmp/hudi"
+    database = "st"
     table_name = "st_test"
+    table_type="COPY_ON_WRITE"
+    record_key_fields="c_bigint"
```

Comment:
Without new config key `op_type`, `database`, `table_type`, `record_key_fields`, can the config execute normally?

Reply:
> Without new config key `op_type`, `database`, `table_type`, `record_key_fields`, can the config execute normally?

Sorry, I did not consider that these item is empty, I have modified it.